src/style.scss

```diff
@@ -13,7 +13,9 @@
   --heading-color: #111111;
   --border-color: #d0d7de;
   --header-bg: white;
-  --block-bg: transparent;
+  --block-bg: white;
+  --header-bg-alternate: #f4f6f7;
```

Comment:
It's preferable to not add a new color var, it's tricky to add a new color because a lot of space have a custom skin and you we don't know by code if its a dark theme (with bg in black and text in white) or if its a light theme. With the alternate color the text may appear white on a white bg, and not be visible, this is an example of skin that would not work https://vote.balancer.fi

Reply:
You could remove a level of nesting for the Transaction Batch to get something more compact this may help remove the need of alternate color to visualize levels. Here is what i mean:
![image](https://user-images.githubusercontent.com/16245250/134826190-54605b72-8657-4151-b80a-d5da2c6078a6.png)

Reply:
Alright, makes sense. I wasn't aware there are custom skins and agree that we must find another way.